internal/querynode/query_collection.go

```diff
@@ -427,6 +428,29 @@ func (q *queryCollection) receiveQueryMsg(msg queryMsg) error {
 	msg.SetTraceCtx(ctx)
 	tr := timerecord.NewTimeRecorder(fmt.Sprintf("receiveQueryMsg %d", msg.ID()))
 
+	// check if timeout
+	curTime := getCurrentTime()
+	//curTimePhysical, _ := tsoutil.ParseTS(curTime)
+	//timeoutTsPhysical, _ := tsoutil.ParseTS(msg.TimeoutTs())
+	//log.Debug("check if query timeout in receiveQueryMsg",
+	//	zap.Any("collectionID", q.collectionID),
+	//	zap.Any("msgID", msg.ID()),
+	//	zap.Any("TimeoutTs", msg.TimeoutTs()),
+	//	zap.Any("curTime", curTime),
+	//	zap.Any("timeoutTsPhysical", timeoutTsPhysical),
+	//	zap.Any("curTimePhysical", curTimePhysical),
+	//)
+	if msg.TimeoutTs() > typeutil.ZeroTimestamp && curTime >= msg.TimeoutTs() {
```

Comment:
redundant code? we should have another internal function named checkTimeout

Reply:
you're right